concent_api/gatekeeper/enum.py

```diff
@@ -0,0 +1,6 @@
+import enum
```

Comment:
The preferred style is to import `Enum` and `unique` explicitly from this module.